<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Dialog{
background-color: rgb(61, 61, 61);}</string>
  </property>
  <widget class="QLabel" name="nome_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>631</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); font-size:36pt</string>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px; 
 background-color: rgb(202, 202, 202);
font-size:11pt;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); font-size:11pt</string>
   </property>
   <property name="text">
    <string>Per cambiare l'immagine del profilo clicca sul bottone sottostante</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gestione_veicoli_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>540</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px; 
 background-color: rgb(202, 202, 202);
font-size:11pt</string>
   </property>
   <property name="text">
    <string>Gestione Veicoli</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gestione_prenotazioni_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>600</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px; 
 background-color: rgb(202, 202, 202);
font-size:11pt</string>
   </property>
   <property name="text">
    <string>Gestione Prenotazioni</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>120</y>
     <width>941</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 61, 61);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="modifica_profilo_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>200</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px; 
 background-color: rgb(202, 202, 202);
font-size:11pt</string>
   </property>
   <property name="text">
    <string>Modifica Informazioni Profilo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cambia_password_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>260</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px; 
 background-color: rgb(202, 202, 202);
font-size:11pt</string>
   </property>
   <property name="text">
    <string>Cambia Password</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>370</y>
     <width>941</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 61, 61);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="telefono_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255); font-size:18pt</string>
      </property>
      <property name="text">
       <string>telefono</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="veicolo_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255); font-size:18pt</string>
      </property>
      <property name="text">
       <string>targa</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>431</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="username_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255); font-size:18pt</string>
      </property>
      <property name="text">
       <string>username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="codice_fiscale_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255); font-size:18pt</string>
      </property>
      <property name="text">
       <string>cf</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="indirizzo_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255); font-size:18pt</string>
      </property>
      <property name="text">
       <string>indirizzo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="email_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255); font-size:18pt</string>
      </property>
      <property name="text">
       <string>email</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
